website/www/site/content/en/roadmap/go-sdk.md

```diff
@@ -17,55 +17,14 @@ limitations under the License.
 
 # Go SDK Roadmap
 
-The Go SDK is currently experimental. As the first purely portable Beam SDK, the Go SDK is constrained
-by the status of the [Beam Portability Framework](https://beam.apache.org/roadmap/portability/) and the existence of
-portable runners.
+The Go SDK is [fully released as of v2.34.0](https://beam.apache.org/blog/go-sdk-release/).
```

Comment:
All your URLs should be relative. (Drop the 'https://beam.apache.org' prefix.)

Reply:
Thanks for the confirmation! I was wondering about that.